rusk-recovery/src/state.rs

```diff
@@ -173,6 +173,10 @@ fn generate_empty_state<P: AsRef<Path>>(
         "../../target/wasm32-unknown-unknown/release/stake_contract.wasm"
     );
 
+    let license_code = include_bytes!(
+        "../../target/wasm32-unknown-unknown/release/license.wasm"
```

Comment:
This need to be renamed in "license-contract" now that you changed the crate name